src/strings.c

```diff
@@ -1911,47 +1911,48 @@ const u8 gText_EVO_Buttons_Decapped_PE[] = _("{DPAD_UPDOWN}Evos  {A_BUTTON}Check
+const u8 gText_EVO_LEVEL[] = _("{LV}{UP_ARROW} to {STR_VAR_2}");
 const u8 gText_EVO_TRADE[] = _("Trading");
 const u8 gText_EVO_TRADE_ITEM[] = _("Trading, holding {STR_VAR_2}");
 const u8 gText_EVO_ITEM[] = _("{STR_VAR_2} is used");
-const u8 gText_EVO_LEVEL_ATK_GT_DEF[] = _("LvlUp to {STR_VAR_2}, Atk > Def");
-const u8 gText_EVO_LEVEL_ATK_EQ_DEF[] = _("LvlUp to {STR_VAR_2}, Atk = Def");
-const u8 gText_EVO_LEVEL_ATK_LT_DEF[] = _("LvlUp to {STR_VAR_2}, Atk < Def");
-const u8 gText_EVO_LEVEL_SILCOON[] = _("LvlUp to {STR_VAR_2}, Silcoon persona");
-const u8 gText_EVO_LEVEL_CASCOON[] = _("LvlUp to {STR_VAR_2}, Cascoon persona");
-const u8 gText_EVO_LEVEL_NINJASK[] = _("LvlUp to {STR_VAR_2}");
-const u8 gText_EVO_LEVEL_SHEDINJA[] = _("LvlUp to {STR_VAR_2}, party<6, 1x POKéBALL");
-const u8 gText_EVO_BEAUTY[] = _("LvlUp, high beauty");
-const u8 gText_EVO_LEVEL_FEMALE[] = _("LvlUp to {STR_VAR_2}, is female");
-const u8 gText_EVO_LEVEL_MALE[] = _("LvlUp to {STR_VAR_2}, is male");
-const u8 gText_EVO_LEVEL_NIGHT[] = _("LvlUp to {STR_VAR_2}, night");
-const u8 gText_EVO_LEVEL_DAY[] = _("LvlUp to {STR_VAR_2}, day");
-const u8 gText_EVO_LEVEL_DUSK[] = _("LvlUp to {STR_VAR_2}, dusk (5-6PM)");
-const u8 gText_EVO_ITEM_HOLD_DAY[] = _("LvlUp, holds {STR_VAR_2}, day");
-const u8 gText_EVO_ITEM_HOLD_NIGHT[] = _("LvlUp, holds {STR_VAR_2}, night");
-const u8 gText_EVO_MOVE[] = _("LvlUp, knows {STR_VAR_2}");
-const u8 gText_EVO_MOVE_TYPE[] = _("LvlUp, knows {STR_VAR_2} move");
-const u8 gText_EVO_MAPSEC[] = _("LvlUp on {STR_VAR_2}");
+const u8 gText_EVO_LEVEL_ATK_GT_DEF[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, Atk > Def");
+const u8 gText_EVO_LEVEL_ATK_EQ_DEF[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, Atk = Def");
+const u8 gText_EVO_LEVEL_ATK_LT_DEF[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, Atk < Def");
+const u8 gText_EVO_LEVEL_SILCOON[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, Silcoon persona");
+const u8 gText_EVO_LEVEL_CASCOON[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, Cascoon persona");
+const u8 gText_EVO_LEVEL_NINJASK[] = _("{LV}{UP_ARROW} to {STR_VAR_2}");
+const u8 gText_EVO_LEVEL_SHEDINJA[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, party<6, 1x POKéBALL");
+const u8 gText_EVO_BEAUTY[] = _("{LV}{UP_ARROW}, high beauty");
+const u8 gText_EVO_LEVEL_FEMALE[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, is female");
+const u8 gText_EVO_LEVEL_MALE[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, is male");
+const u8 gText_EVO_LEVEL_NIGHT[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, night");
+const u8 gText_EVO_LEVEL_DAY[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, day");
+const u8 gText_EVO_LEVEL_DUSK[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, dusk (5-6PM)");
+const u8 gText_EVO_ITEM_HOLD_DAY[] = _("{LV}{UP_ARROW}, holds {STR_VAR_2}, day");
+const u8 gText_EVO_ITEM_HOLD_NIGHT[] = _("{LV}{UP_ARROW}, holds {STR_VAR_2}, night");
+const u8 gText_EVO_MOVE[] = _("{LV}{UP_ARROW}, knows {STR_VAR_2}");
+const u8 gText_EVO_MOVE_TYPE[] = _("{LV}{UP_ARROW}, knows {STR_VAR_2} move");
```

Comment:
You didn't remove `gText_EVO_MOVE_TYPE`

Reply:
Oh heck my bad, just pushed a correction